### ISSUE TITLE
nvda lags when navigating document in microsoft word with braille display turned on

### ISSUE THREAD

**Issue by zstanecic** [nvaccess/nvda#6970]
Operating system: windows 10
microsoft office 2016,
Handytech braille wave as a braille display,
when navigating a word document with a braille display, NVDA lags for a second. especially when navigating larger documents like 27 page documents or more. That’s also the problem with other displays, confirmed when talking with one friend wih has focus braille display. Theres’ no error, it’s lag wich is not in log.

**Comment by Adriani90** [comment]
@zstanecic the performance of NVDA in MS Word improved significantly since this issue has been reported. Are you still having this problem in NVDA 2019.1?

**Comment by Adriani90** [comment]
@zstanecic are you still available to test with last NVDA alpha if this issue is still significant? I should have been improved alot meanwhile.

**Comment by Adriani90** [comment]
Closing in favor of #11089 which is newer and indicates the same issue, lower performance and even crash.
